synapse/lib/hive.py

```diff
@@ -220,6 +220,9 @@ def onedit(self, path, func, base=None):
 
         return node.valu
 
+    async def exists(self, full):
+        return full in self.nodes
```

Comment:
It would be nice to see this tested in test_lib_hive

Reply:
Done